src/python/WMCore/Storage/StageOutMgr.py

```diff
@@ -99,65 +135,55 @@ def initialiseSiteConf(self):
         """
         _initialiseSiteConf_
 
-        Extract required information from site conf and TFC
+        Extract required information from site conf and RFC
 
         """
-
-        implName = self.siteCfg.localStageOut.get("command", None)
-        if implName == None:
-            msg = "Unable to retrieve local stage out command\n"
-            msg += "From site config file.\n"
-            msg += "Unable to perform StageOut operation"
-            raise StageOutInitError(msg)
-        msg = "Local Stage Out Implementation to be used is: "
-        msg += "%s\n" % implName
-
-        pnn = self.siteCfg.localStageOut.get("phedex-node", None)
-        if pnn == None:
-            msg = "Unable to retrieve local stage out phedex-node\n"
-            msg += "From site config file.\n"
-            msg += "Unable to perform StageOut operation"
-            raise StageOutInitError(msg)
-        msg += "Local Stage Out PNN to be used is %s\n" % pnn
-        catalog = self.siteCfg.localStageOut.get("catalog", None)
-        if catalog == None:
-            msg = "Unable to retrieve local stage out catalog\n"
-            msg += "From site config file.\n"
-            msg += "Unable to perform StageOut operation"
-            raise StageOutInitError(msg)
-        msg += "Local Stage Out Catalog to be used is %s\n" % catalog
-
-        try:
-            self.tfc = self.siteCfg.trivialFileCatalog()
-            msg += "Trivial File Catalog has been loaded:\n"
-            msg += str(self.tfc)
-        except Exception as ex:
-            msg = "Unable to load Trivial File Catalog:\n"
-            msg += "Local stage out will not be attempted\n"
-            msg += str(ex)
+        self.stageOuts = self.siteCfg.stageOuts
+
+        msg = "\nThere are %s stage out definitions." % len(self.stageOuts)
+        for stageOut in self.stageOuts:
+            for k in ['phedex-node','command','storageSite','volume','protocol']:
```

Comment:
We should probably define these attributes as constants somewhere, given that they are used in multiple places.

Reply:
These are used three times in DeleteMgr.py, StageOutMrg.py and StageInMgr.py